.github/workflows/verify.yml

```diff
@@ -4,10 +4,6 @@ on:
   pull_request:
     branches-ignore:
       - 'changeset-release/**'
-  push:
-    branches:
-      - main
-      - 'release/**'
```

Comment:
This was actually the culprit for why status checks weren't running. There is really no reason to filter on this.